.travis.yml

```diff
@@ -48,6 +48,10 @@ install:
     - pip install -r requirements.txt
     - pip install -r requirements_test.txt
 
+    # image gradients plug-ins
+    - pip install plugins/data/imageGradients
+    - pip install plugins/view/imageGradients
```

Comment:
So... you don't actually have to `pip install` DIGITS? You only have to install the plugins?